pkgs/applications/window-managers/windowchef/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, libxcb, libXrandr
+, xcbutil, xcbutilkeysyms, xcbutilwm, xcbproto
+}:
+
+stdenv.mkDerivation rec {
+  pname = "windowchef";
+  version = "v0.5.0";
+
+  src = fetchFromGitHub {
+    owner  = "tudurom";
+    repo   = "windowchef";
+    rev    = version;
```

```suggestion
    rev    = "v${version}";
```

As a corollary to the above.

---

I initially had just the version but it doesn't find the tar.gz package (the url string include the prefix of v.  Is there a better place to put this?

---

Since `fetchFromGitHub` takes the tarball from the tag, this shouldn't matter. If you're dealing with a raw URL, though, you can do as I did here: `v${version}`, e.g.:

`https://github.com/ajeetdsouza/zoxide/archive/v${version}.tar.gz`

Whatever you need to not have any letters in the package version.